### ISSUE TITLE
[BUG] nltk download stopwords fails

### ISSUE THREAD

**Issue by malharjere** [robustness-gym/robustness-gym#8]
**Describe the bug**
I tried to import robustnessgym in a virtual environment in python 3.8.7 but wasn't able to. I installed it with pip, and got the following error:

```
Python 3.8.7 (v3.8.7:6503f05dd5, Dec 21 2020, 12:45:15)
[Clang 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import robustnessgym
Traceback (most recent call last):
  File "/Users/malharjere/Desktop/robustnessgym/lib/python3.8/site-packages/nltk/corpus/util.py", line 83, in __load
    root = nltk.data.find("{}/{}".format(self.subdir, zip_name))
  File "/Users/malharjere/Desktop/robustnessgym/lib/python3.8/site-packages/nltk/data.py", line 585, in find
    raise LookupError(resource_not_found)
LookupError:
**********************************************************************
  Resource stopwords not found.
  Please use the NLTK Downloader to obtain the resource:

  >>> import nltk
  >>> nltk.download('stopwords')

  For more information see: https://www.nltk.org/data.html

  Attempted to load corpora/stopwords.zip/stopwords/

  Searched in:
    - '/Users/malharjere/nltk_data'
    - '/Users/malharjere/Desktop/robustnessgym/nltk_data'
    - '/Users/malharjere/Desktop/robustnessgym/share/nltk_data'
    - '/Users/malharjere/Desktop/robustnessgym/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/malharjere/Desktop/robustnessgym/lib/python3.8/site-packages/robustnessgym/__init__.py", line 26, in <module>
    from robustnessgym.slicebuilders.attacks.textattack import TextAttack
  File "/Users/malharjere/Desktop/robustnessgym/lib/python3.8/site-packages/robustnessgym/slicebuilders/attacks/__init__.py", line 2, in <module>
    from robustnessgym.slicebuilders.attacks.textattack import TextAttack  # noqa
  File "/Users/malharjere/Desktop/robustnessgym/lib/python3.8/site-packages/robustnessgym/slicebuilders/attacks/textattack.py", line 6, in <module>
    import textattack.attack_recipes as attack_recipes
  File "/Users/malharjere/Desktop/robustnessgym/lib/python3.8/site-packages/textattack/__init__.py", line 12, in <module>
    from . import (
  File "/Users/malharjere/Desktop/robustnessgym/lib/python3.8/site-packages/textattack/augmentation/__init__.py", line 9, in <module>
    from .recipes import (
  File "/Users/malharjere/Desktop/robustnessgym/lib/python3.8/site-packages/textattack/augmentation/recipes.py", line 18, in <module>
    DEFAULT_CONSTRAINTS = [RepeatModification(), StopwordModification()]
  File "/Users/malharjere/Desktop/robustnessgym/lib/python3.8/site-packages/textattack/constraints/pre_transformation/stopword_modification.py", line 21, in __init__
    self.stopwords = set(nltk.corpus.stopwords.words("english"))
  File "/Users/malharjere/Desktop/robustnessgym/lib/python3.8/site-packages/nltk/corpus/util.py", line 120, in __getattr__
    self.__load()
  File "/Users/malharjere/Desktop/robustnessgym/lib/python3.8/site-packages/nltk/corpus/util.py", line 85, in __load
    raise e
  File "/Users/malharjere/Desktop/robustnessgym/lib/python3.8/site-packages/nltk/corpus/util.py", line 80, in __load
    root = nltk.data.find("{}/{}".format(self.subdir, self.__name))
  File "/Users/malharjere/Desktop/robustnessgym/lib/python3.8/site-packages/nltk/data.py", line 585, in find
    raise LookupError(resource_not_found)
LookupError:
**********************************************************************
  Resource stopwords not found.
  Please use the NLTK Downloader to obtain the resource:

  >>> import nltk
  >>> nltk.download('stopwords')

  For more information see: https://www.nltk.org/data.html

  Attempted to load corpora/stopwords

  Searched in:
    - '/Users/malharjere/nltk_data'
    - '/Users/malharjere/Desktop/robustnessgym/nltk_data'
    - '/Users/malharjere/Desktop/robustnessgym/share/nltk_data'
    - '/Users/malharjere/Desktop/robustnessgym/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************
```

**System Information**
 - OS: MacOS
 - `robustnessgym` (v0.0.2)

**Comment by krandiash** [comment]
This seems like an `nltk` error: try running `python -m nltk.downloader stopwords` in command line and then try the import again.

**Comment by malharjere** [comment]
Did that, got this:

```(robustnessgym) FiddlerITs-MBP:robustnessgym malharjere$ python -m nltk.downloader stopwords
/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/runpy.py:127: RuntimeWarning: 'nltk.downloader' found in sys.modules after import of package 'nltk', but prior to execution of 'nltk.downloader'; this may result in unpredictable behaviour
  warn(RuntimeWarning(msg))
[nltk_data] Error loading stopwords: <urlopen error [SSL:
[nltk_data]     CERTIFICATE_VERIFY_FAILED] certificate verify failed:
[nltk_data]     unable to get local issuer certificate (_ssl.c:1125)>
Error installing package. Retry? [n/y/e]
y
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/malharjere/Desktop/robustnessgym/lib/python3.8/site-packages/nltk/downloader.py", line 2533, in <module>
    rv = downloader.download(
  File "/Users/malharjere/Desktop/robustnessgym/lib/python3.8/site-packages/nltk/downloader.py", line 793, in download
    msg.package.id,
AttributeError: 'NoneType' object has no attribute 'id'
```

**Comment by malharjere** [comment]
Fixed with this, replaced Python 3.6 with 3.8: https://stackoverflow.com/questions/41348621/ssl-error-downloading-nltk-data
